lib/react.js

```diff
@@ -1,4394 +1,301 @@
nt);\n    }\n    componentDidMount() {\n      dartInteropStatics.handleComponentDidMount(this.dartComponent);\n    }\n    /*\n    /// This cannot be used with UNSAFE_ lifecycle methods.\n    getDerivedStateFromProps(nextProps, prevState) {\n      return dartInteropStatics.handleGetDerivedStateFromProps(this.props.internal, nextProps.internal);\n    }\n    */\n    UNSAFE_componentWillReceiveProps(nextProps, nextContext) {\n      dartInteropStatics.handleComponentWillReceiveProps(this.dartComponent, nextProps.internal, nextContext);\n    }\n    shouldComponentUpdate(nextProps, nextState, nextContext) {\n      return dartInteropStatics.handleShouldComponentUpdate(this.dartComponent, nextContext);\n    }\n    /*\n    /// This cannot be used with UNSAFE_ lifecycle methods.\n    getSnapshotBeforeUpdate() {\n      return dartInteropStatics.handleGetSnapshotBeforeUpdate(this.props.internal, prevProps.internal);\n    }\n    */\n    UNSAFE_componentWillUpdate(nextProps, nextState, nextContext) {\n      dartInteropStatics.handleComponentWillUpdate(this.dartComponent, nextContext);\n    }\n    componentDidUpdate(prevProps, prevState) {\n      dartInteropStatics.handleComponentDidUpdate(this.dartComponent, prevProps.internal);\n    }\n    componentWillUnmount() {\n      dartInteropStatics.handleComponentWillUnmount(this.dartComponent);\n    }\n    render() {\n      var result = dartInteropStatics.handleRender(this.dartComponent);\n      if (typeof result === 'undefined') result = null;\n      return result;\n    }\n  }\n\n      // React limits the accessible context entries\n      // to the keys specified in childContextTypes/contextTypes.\n      var childContextKeys = jsConfig && jsConfig.childContextKeys;\n      var contextKeys = jsConfig && jsConfig.contextKeys;\n\n      if (childContextKeys && childContextKeys.length !== 0) {\n        ReactDartComponent.childContextTypes = {};\n        for (var i = 0; i < childContextKeys.length; i++) {\n          ReactDartComponent.childContextTypes[childContextKeys[i]] = React.PropTypes.object;\n        }\n        // Only declare this when `childContextKeys` is non-empty to avoid unnecessarily\n        // creating interop context objects for components that won't use it.\n        ReactDartComponent.prototype['getChildContext'] = function() {\n          return dartInteropStatics.handleGetChildContext(this.dartComponent);\n        };\n      }\n\n      if (contextKeys && contextKeys.length !== 0) {\n        ReactDartComponent.contextTypes = {};\n        for (var i = 0; i < contextKeys.length; i++) {\n          ReactDartComponent.contextTypes[contextKeys[i]] = React.PropTypes.object;\n        }\n      }\n\n  return ReactDartComponent;\n}\n\nfunction _markChildValidated(child) {\n  const store = child._store;\n  if (store) store.validated = true;\n}\n\nmodule.exports = {\n  _getProperty,\n  _setProperty,\n  _createReactDartComponentClass,\n  _markChildValidated,\n};\n\n\n//# sourceURL=webpack:///./js_src/_dart_helpers.js?");
```

Comment:
We can't use `eval` in any of our deployed environments due to CSP.

While most of the time we'll be deploying prod React JS, there are cases where the development version would be useful to pull in. Can we configure webpack to not use eval?

Reply:
Ill check it out.